signingscript/docker.d/passwords.yml

```diff
@@ -202,6 +202,12 @@ in:
              {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
              ["autograph_apk"]
             ]
+        project:mobile:fenix:releng:signing:cert:fennec-nightly-signing:
```

Comment:
Should we remove the trailing `:`?

Reply:
That's a good question. I see the other entries have it too. Is it causing any trouble?

Reply:
It may cause confusion when we add it to the restricted scopes [here](https://github.com/escapewindow/scriptworker/blob/master/src/scriptworker/constants.py#L389-L404).

Reply:
Okay. As far as I understand, the trailing comma indicates `project:mobile:fenix:releng:signing:cert:fennec-nightly-signing` is a key of a YAML object defined here https://github.com/mozilla-releng/scriptworker-scripts/blob/c94fc477ddddbb1984ea8f6c0d93398980c92157/signingscript/docker.d/passwords.yml#L14.

I'm sorry, I'm not too sure to understand where the confusion may come from.